learning/scripting/visual_script/visual_script_basics.rst

```diff
@@ -84,7 +89,7 @@ In Godot, the scripting interface is universal and all implementations conform t
 
 A function is an individual canvas with nodes connected.
 
-A single script can contain many functions, each of which will have a canvas of its own. This makes scripts
+A single script can contain many functions, each of which will have a canvas of its own, making scripts
```

Comment:
This doesn't inherently make things more organized. It allows the programmer more options for organization, but it doesn't happen automatically.  "own, allowing for more organization."